department-of-veterans-affairs/caseflow
# 8776 update add decisions page

Resolves #8778 and #8776

### Description
Made the changes described in this ticket:
https://github.com/department-of-veterans-affairs/caseflow/issues/8778

### Acceptance Criteria

### Testing Plan
1. run `bundle exec rails runner scripts/enable_features_dev.rb`
2.  login as judge or attorney
3. select AMA appeal
4. select decision ready for review
5. no special issues
6. on add decisions page, add a decision to any issue
7. verify a delete option now appears next to the edit button:
<img width="1360" alt="screen shot 2019-01-18 at 3 35 19 pm" src="https://user-images.githubusercontent.com/46791771/51411688-c5123700-1b36-11e9-8871-3480a9221a79.png">
8. on clicking delete, verify this modal appears and that the 'cancel' and 'Yes, delete decision' buttons work:
<img width="589" alt="screen shot 2019-01-18 at 3 36 21 pm" src="https://user-images.githubusercontent.com/46791771/51411710-deb37e80-1b36-11e9-84d5-e303e6f2aa26.png">
9. verify that the appropriate text appears if there are connected issues associated with the issue you are trying to delete. (you can check if there are connected issues by seeing if there is a 'select issues' dropdown when you go to add an issue)
